pkg/cluster/px/px.go

```diff
@@ -198,8 +211,23 @@ func (ops *pxClusterOps) Upgrade(new *apiv1alpha1.Cluster, opts *UpgradeOptions)
 	return nil
 }
 
-func (ops *pxClusterOps) Destroy(c *apiv1alpha1.Cluster) error {
-	logrus.Warnf("destroying px cluster is implemented")
+func (ops *pxClusterOps) Delete(c *apiv1alpha1.Cluster, opts *DeleteOptions) error {
+	if opts != nil && opts.WipeCluster {
+		// the wipe operation is best effort as it is used when cluster is already in a bad shape. for e.g
+		// cluster might have been started with incorrect kvdb information. So we won't be able to wipe that off
+
+		// 1. Cleanup PX kvdb tree
+		err := ops.wipePXKvdb()
+		if err != nil {
+			logrus.Warnf("failed to wipe Portworx KVDB tree. err: %v", err)
+		}
+
+		// 2. TODO remove PX systemd service on each node
```

Comment:
Do we want to remove the systemd service?
Most of the times we will want to just fix the nodes as there was a failed install and restart portworx service.

We will have to either delete the whole OCI installation and remove the systemd service or do nothing.

Reply:
https://github.com/portworx/talisman/issues/31 will delete the whole OCI installation